File: starter-bots/JavaBot/src/main/java/Models/GameStateDto.java
package Models;

import java.util.*;

public class GameStateDto {

  private World world;
  private Map<String, List<Integer>> gameObjects;
  private Map<String, List<Integer>> playerObjects;

  public Models.World getWorld() {
    return world;
  }

  public void setWorld(Models.World world) {
    this.world = world;
  }

  public Map<String, List<Integer>> getGameObjects() {
    return gameObjects;
  }

  public void setGameObjects(Map<String, List<Integer>> gameObjects) {
    this.gameObjects = gameObjects;
  }

  public Map<String, List<Integer>> getPlayerObjects() {
    return playerObjects;
  }

  public void setPlayerObjects(Map<String, List<Integer>> playerObjects) {
    playerObjects = playerObjects;
  }
}

File: starter-bots/ReferenceBot/ReferenceBot/Models/GameStateDto.cs
using System.Collections.Generic;

namespace NETCoreBot.Models
{
    public class GameStateDto
    {
        public World World { get; set; }
        public Dictionary<string, List<int>> GameObjects { get; set; }
        public Dictionary<string, List<int>> PlayerObjects { get; set; }
    }
}